apps/geoportal/src/app/components/HomeButton.tsx

```diff
@@ -0,0 +1,51 @@
+import { useContext, useEffect } from 'react';
+import { TopicMapContext } from 'react-cismap/contexts/TopicMapContextProvider';
+import L from 'leaflet';
+import 'leaflet/dist/leaflet.css';
+
+const HomeButton = () => {
```

Comment:
@d4v3000 @PavelOlkhovoi  could this Home functionality be replaced with something from https://github.com/cismet/carma/tree/dev/libraries/mapping/carma-map-control-layout/src/lib/components already?
Or would that need to be added?